src/ffi_gen_tmp.h

```diff
@@ -25,7 +25,7 @@ struct CaptureDevice
 
 
 
-FFI_PLUGIN_EXPORT float getFxParams(enum FilterType filterType, int attributeId);
-
-
-
+/// Get visualization state
```

Comment:
Just checking that you meant to remove `getFxParams` from here.

Reply:
Nothing removed :)
This file is just temporary and it is not used by any other sources.